docs/reference.markdown

```diff
@@ -129,7 +131,9 @@ The following entries are optional in the header:
       ...
     ```
 
-    The specified dependencies will be built before building the given package.
+    The specified dependencies will be built before building the given package; 
+    the "required" corresponding binaries, libraries and environment variables 
+    will be made available (by loading the dependency module) before the current built starts.
```

Comment:
Do you confirm that a `requires` or `build_requires` load the dependency module before the build starts ?
I mean, it is a "casual" loading of module, there is nothing extra or specific (bin, lib, env) due to the fact that we are about to compile a new package ? In other words, loading a module, be it for a coming built-time or run-time, is the same ?
(I was not sure about this).

Reply:
not the dependency module, they load the build environment of the dependencies. aliBuild has no idea about modulefiles.